Client/Frontend/Settings/NTPTableViewController.swift

```diff
@@ -94,14 +109,10 @@ class NTPTableViewController: TableViewController {
         }
         return row
     }
-    
-    private lazy var section: Section = {
-        var section = Section(rows: [])
-        section.rows = [
-            Row.boolRow(title: Strings.NTP.settingsAutoOpenKeyboard, option: Preferences.NewTabPage.autoOpenKeyboard),
-            Row.boolRow(title: Strings.NTP.settingsBackgroundImages, option: Preferences.NewTabPage.backgroundImages),
-            moreSetting(section: section)
-        ]
-        return section
-    }()
+}
+
+extension NTPTableViewController: PreferencesObserver {
+    func preferencesDidChange(for key: String) {
+        loadSections()
```

Comment:
This is unfortunate, but the `Static` framework isn't super flexible when it comes to dynamically updating rows, and there's no provided method to disable the row or hide it.